src/Groups/GAPGroups.jl

```diff
@@ -1701,6 +1701,40 @@ function length(g::FPGroupElem)
 end
 
 
+"""
+    (G::FPGroup)(l::AbstractVector{Pair{T, S}}) where {T <: IntegerUnion, S <: IntegerUnion}
+
+Return the element `x` of `G` that is described by `l`
```

Comment:
The font used by my browser for GitHub render `l` quite similar to `i1j`. Perhaps we could use a slightly more distinctive name here, e.g. `list` or `pairs` or so